src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +543,44 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
+
+			var fileName = localPath;
+			var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1) fileName = fileName.Substring(pos + 1);
+
+			// try to find the css file
+			var path = CurrentBook.GetFileLocator().LocateFile(fileName);
+
+			// we want these stylesheets to come from the book folder
+			if (path.Contains("languageDisplay.css"))
```

Comment:
If languageDisplay.css IS in the book folder, and (I think now) the html simply has href="languageDisplay.css", then (since we're generating page files with a trick that makes them seem to be in the book folder) the browser will look for it using a url like http://localhost:8089/bloom/c:/.../someCollection/somebook/languageDisplay.css. That's exactly where it is, so a preliminary check for file.Exists(localPath) would find it without any knowledge of the current book.
If we made the href for CollectionStyles.css be "../CollectionStyles.css" the same trick would work for it (unless cross-domain stuff messes us up?) and we might not need to know the current book at all.
(I've also wondered whether we should be copying CollectionStyles.css into the book folder so we don't hit the cross-domain thing if someone just opens the HTML file, but that's another issue.)